e2e/sidecars/sidecars_test.go

```diff
@@ -49,14 +49,18 @@ sidecars:
   nginx:
     port: 80
     image: %s    # Image URL for sidecar container.
+    variables:
+      NGINX_PORT: %s
 logging:
   destination:
     Name: cloudwatch
-    region: us-west-2
+    region: us-east-1
```

Comment:
I'm just curious why you changed regions here...?

Reply:
good question it is because our e2e tests are deployed in [`us-east-1`](https://github.com/aws/copilot-cli/blob/1a5c46e0f2a8d6a53962ce502db347208c71f3b2/.release/buildspec_e2e.yml#L75), so that firelens knows where to find the log group and create the log stream in it.

Reply:
but i do think we need to update the e2e test so that this error can be caught.

Reply:
ok now it should be able to catch it if the firelens is not able to create the log stream in the log group.